packages/nimble-components/src/patterns/expand-collapse/styles.ts

```diff
@@ -0,0 +1,34 @@
+import { css } from '@microsoft/fast-element';
+import {
+    controlSlimHeight,
+    mediumDelay
+} from '../../theme-provider/design-tokens';
+
+export const styles = css`
+    :host {
+        --ni-private-expand-collapse-button-indent-level: 0px;
+    }
+
+    :host .animating {
+        transition: ${mediumDelay} ease-in-out;
```

Comment:
How closely should we align this with [the tree item styles](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/tree-item/styles.ts)? Some differences I noticed 
1. tree uses `ease-in`, not `ease-in-out`
2. tree applies a `180deg` rotation while this uses `0deg` (haven't dug into how that manifests) 
3. tree supports rtl but this doesn't
4. similar but not identical names and purposes of private CSS variables

I don't think they necessarily have to align, especially if the DOM structure is not the same. But it'd be worth a pass to see if the tree is doing anything that this should copy. I doubt they'd share a pattern but maybe they could.

Reply:
Yes, as I mention in the description, sharing this style with the `TreeItem` would certainly be ideal, but there are some differences that prevent that outright. That said, we can certainly address some of the differences you outline (though I would prefer to do that as a separate PR).

Reply:
I've updated to use `ease-in` for consistency with `Treee`. For now, I don't think there's any other sharing to be done, or consistency to enforce.